src/modules/dashboard/components/FeaturedChannels.tsx

```diff
@@ -47,66 +30,18 @@ const FeaturedChannels: FC<FeaturedChannelsProps> = () => {
       backgroundColor={{ light: 'darkWhite', dark: 'gray-900' }}
       gap={{ ml: 's4', initial: 's2' }}
     >
-      <Box
-        display="flex"
-        justifyContent="space-between"
-        flexDirection={{ tb: 'column' }}
-        gap={{ tb: 's3' }}
-      >
-        <Text
-          variant="h4-bold"
-          color={{ light: 'black', dark: 'white' }}
-        >
-          Featured Notification Channels
-        </Text>
-
-        <Box
-          display="flex"
-          flexDirection="row"
-          alignItems="center"
-          gap="s4"
-        >
-          <Link
-            to="/channels"
-            textProps={{
-              variant: 'h5-semibold',
-              color: { light: 'black', dark: 'white' },
-            }}
-          >
-            View All
-          </Link>
-
-          <Box
-            display="flex"
-            flexDirection="row"
-          >
-            <HoverableSVG
-              onClick={handlePrevious}
-              defaultColor={{ light: 'gray-900', dark: 'gray-400' }}
-              disabled={currentIndex === 0}
-              icon={<PrevIconSlider size={24} />}
-            ></HoverableSVG>
-            <HoverableSVG
-              onClick={handleClick}
-              defaultColor={{ light: 'gray-900', dark: 'gray-400' }}
-              disabled={currentIndex + itemsPerPage >= applicableFeaturedChannels.length}
-              icon={<NextIconSlider size={24} />}
-            ></HoverableSVG>
-          </Box>
-        </Box>
-      </Box>
 
       {showMobileAndTabletView ? (
         <FeaturedChannelsMobileViewList
-          listRef={listRef}
           featuredChannelsList={mobileFeaturedChannelsList}
         />
+
       ) : (
         <FeaturedChannelsList
-          listRef={listRef}
           featuredChannelsList={featureChannelsForCurrrentEnv}
         />
       )}
+
```

Comment:
I think the FeaturedChannelsList.tsx and FeaturedChannelsMobileView.tsx can be combined into one component called  FeaturedChannelsList.tsx for mobile small, mobile medium, mobile large, laptop, laptop large and desktop screen sizes.

We might need a separate FeaturedChannelsTabletView.tsx component. Please see what approach works best.

Reply:
FeaturedChannels is divided into two components. FeaturedChannels and FeaturedChannelsMobileView. 

Laptop and tablet will be rendered into FeaturedChannels and mobile will be rendered with FeaturedChannelsMobile. 

In the tablet mode, 2 slides will be shown and on clicking next 2 slides will be shown. In the laptop mode, 3 slides will be shown and on clicking next 3 slides will be shown.

In the mobile view, 3 channels are rendered on a single viewport vertically and upon clicking next, next slide is shown.